tests/subprocesstest.py

```diff
@@ -84,15 +84,16 @@ def _run_test_in_subprocess(self, result):
             stderr=subprocess.PIPE,
             env=sp_test_env,
         )
-        _, stderr = sp.communicate()
+        stdout, stderr = sp.communicate()
 
         if sp.returncode:
             try:
                 cmdf = ' '.join(sp_test_cmd)
                 raise Exception('Subprocess Test "{}" Failed'.format(cmdf))
             except Exception:
                 exc_info = sys.exc_info()
-            sys.stderr.write(stderr)
+            sys.stderr.write(stderr.decode())
+            sys.stdout.write(stdout.decode())
```

Comment:
These provide a fix for Python 3 where the output may be bytes.

We also now output the stdout as well so the test runner can handle it.

Reply:
This isn't going to be an issue for Python2 at all is it?

Reply:
Should we be using a `compat.stringify` or something here?

Reply:
It seems to work, but I'll see if `stringify` works and use that if so.

Reply:
`compat.stringify` for python 3 prints the byte string which isn't very friendly for reading:

```
.b'F\n======================================================================\nFAIL: test_deregister_post_import_hook_after_register_multiple_one_match (tests.utils.test_hook.TestHook)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/Users/kyle.verhoog/dev/dd-trace-py/tests/utils/test_hook.py", line 144, in test_deregister_post_import_hook_after_register_multiple_one_match\n    self.assertEqual(test_hook2.call_count, 1, \'hook should have been called\')\nAssertionError: 0 != 1 : hook should have been called\n\n----------------------------------------------------------------------\nRan 1 test in 0.001s\n\nFAILED (failures=1)\n'b''
```

It seems to work as is for python 2 and 3, so I think we're okay.

Reply:
great